src/Renderer/WebXR.js

```diff
@@ -0,0 +1,72 @@
+/* global XRRigidTransform */
+
+import * as THREE from 'three';
+import { VRButton } from 'ThreeExtended/webxr/VRButton';
+
+async function shutdownXR(session) {
+    if (session) {
+        await session.end();
+    }
+}
+
+const initializeWebXR = (view, options) => {
+    document.body.appendChild(VRButton.createButton(view.mainLoop.gfxEngine.renderer));
+    const xr = view.mainLoop.gfxEngine.renderer.xr;
+    const scale = options.scale || 1.0;
+
+    xr.addEventListener('sessionstart', () => {
+        console.log('Web XR session start'); // eslint-disable-line
+
+        const camera = view.camera.camera3D;
+        const webXRManager = view.mainLoop.gfxEngine.renderer.xr;
+
+        const exitXRSession =  (event) => {
+            if (event.key === 'Escape') {
+                console.log('Web XR session stop'); // eslint-disable-line
+                document.removeEventListener('keydown', exitXRSession);
+                view.mainLoop.gfxEngine.renderer.xr.enabled = false;
+                view.camera.camera3D = camera;
+
+                view.scene.scale.multiplyScalar(1 / scale);
+                view.scene.updateMatrixWorld();
+
+                shutdownXR(webXRManager.getSession());
+                view.notifyChange(view.camera.camera3D, true);
+            }
+        };
+        view.scene.scale.multiplyScalar(scale);
+        view.scene.updateMatrixWorld();
+        view.mainLoop.gfxEngine.renderer.xr.enabled = true;
+        view.mainLoop.gfxEngine.renderer.xr.getReferenceSpace('local');
```

Comment:
Can you use `xr` or `webXRManager` constant instead ? Or else remove these constants which is also fine by me ?

Reply:
Done.